.Rbuildignore

```diff
@@ -12,4 +12,20 @@
 ^JUNK
 ^LICENSE\.md$
 ^\.gitignore$
+^\.github$
+
 ^CODE_DESIGN.md$
+^README.Rmd$
+^RELEASE.R$
+^data-raw$
+
+
+# Temp omit for checking
+# ^R/Add_points.R$
```

Comment:
Not sure what best procedure is here. Should I uncomment these before doing a release or leave as is? Some look like they can stay ignored, but others are still useful.